blocklist.sh

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: (c) 2024 Ring Zero Desenvolvimento de Software LTDA
+# SPDX-License-Identifier: GPL-2.0-only
+#!/bin/sh
+
+url="https://blocklistproject.github.io/Lists/alt-version"
+dir="blocklist"
+lists="abuse ads crypto drugs facebook fraud gambling malware phishing\
+	piracy porn ransomware redirect scam tiktok torrent tracking"
+
+mkdir -p ${dir}
+for name in ${lists}; do
+	file="${name}-nl.txt"
+	list="${dir}/${file}"
+	wget --show-progress -q "${url}/${file}" -O ${list}
```

Comment:
```
sav@alekhine:~$ wget --show-progress
Unknown option 'show-progress'
sav@alekhine:~$ wget --version
GNU Wget2 2.1.0 - multithreaded metalink/file/website downloader

+digest +https +ssl/gnutls +ipv6 +iri +large-file +nls -ntlm -opie +psl -hsts
+iconv +idn2 +zlib -lzma +brotlidec +zstd -bzip2 -lzip +http2 +gpgme

Copyright (C) 2012-2015 Tim Ruehsen
Copyright (C) 2015-2021 Free Software Foundation, Inc.

License GPLv3+: GNU GPL version 3 or later
<http://www.gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Please send bug reports and questions to <bug-wget@gnu.org>.
```

Reply:
it's a Fedora 40 (latest).

Reply:
can you try `wget -q --show-progress --progress=bar:force:noscroll`?

Reply:
nevermind.. it changed to `--force-progress` on wget2.. will change it to curl, even having a uglier progress bar.. I think it's better to be more portable

Reply:
works for me :)